elastic-stack/elasticsearch/plugins.sls

```diff
@@ -1,15 +1,19 @@
 {% from "elastic-stack/elasticsearch/map.jinja" import elasticsearch with context %}
-{% set commands = {'install': 'install -b', 'remove': 'remove'} %}
 
 include:
   - .service
+  - .install
 
 {% for plugin in salt.pillar.get('elastic_stack:elasticsearch:plugins', []) %}
-{% for command in commands %}
-{{ command }}_{{ plugin.name }}_plugin:
+install_{{ plugin.name }}_plugin:
```

Comment:
So, I would split this out into separate state IDs, e.g.
```
remove_{{ plugin.name }}_plugin:
    cmd.run:
      - name: <remove plugin>
      - onchanges:
         - pkg: install_elasticsearch
      - require_in:
        - cmd: install_{{ plugin.name }}_plugin

install_{{ plugin.name }}_plugin:
   cmd.run:
 ....
```

Reply:
I've split this in commit 9810115602666cc33854550c7010353dfe7c1687